src/eHealth.couchQuery/services/lucene-query-factory.js

```diff
@@ -104,9 +105,12 @@ angular
                   .then(function(response) {
                     return response.data;
                   });
-              }, initialParams);
+              }, initialParams, initialOptions);
             } else {
               initialParams.q = q;
+              if (!initialOptions.hasOwnProperty('unique')) {
```

Comment:
instead of an `if` here, i would prefer:

    initialOptions = angular.extend({
      unique: true,
    }, initialOptions);

At line 92. More declarative and it scales better with the number of default initial options